malaria-ios/Managers/UserSettingsManager.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+class UserSettingsManager{
+    
+    /// Sets an UserSetting
+    /// :param: `UserSetting`: the setting to be changed
+    /// :param: `Bool`: value
+    class func setBool(key: UserSetting, _ value: Bool){
```

Comment:
Yeah, it might seem a bit redundant but the benefit is that you can easily see all the different user defaults that your system is using (with autocomplete :+1: ), and also all the keys you're using so there won't be any overlaps.  It solves the same concern as the `UserSetting` you had in consolidating the strings in one place, but with one less layer of abstraction.